.env

```diff
@@ -4,5 +4,6 @@ REACT_APP_TITLE=Tailscale
 REACT_APP_DESCRIPTION=Status page for Tailscale
 REACT_APP_LOGO=https://raw.githubusercontent.com/tailscale/issue-status/master/.issuestatus/tailscale.png
 REACT_APP_NAME=Tailscale Status
+REACT_APP_LINK=https://tailscale.com
```

Comment:
Including it as an env variable is fine. You could probably also hard code it if you wanted, since I don't think it's changing any time soon 😄

Reply:
Yeah, I made this an env variable so I can upstream this patch; I assume other users of this may want a link in the header :smiley: